Lesson10/src/Main.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Scanner;
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Technically possible, but sense was in array of different objects (cats and dogs in one storage)